docs/pages/index.mdx

```diff
@@ -28,6 +30,10 @@ import jamfProSvg from "@site/src/components/Icon/teleport-svg/jamf-pro.svg";
 import jenkinsSvg from "@site/src/components/Icon/teleport-svg/jenkins.svg";
 import snowflakeSvg from "@site/src/components/Icon/teleport-svg/snowflake.svg";
 import arrowRightSvg from "@site/src/components/Icon/teleport-svg/arrow-right.svg";
+import zeroTrustSvg from "@site/src/components/Icon/teleport-svg/zero-trust.svg";
+import MWISvg from "@site/src/components/Icon/teleport-svg/mwi.svg";
+import IGSvg from "@site/src/components/Icon/teleport-svg/identity-governance.svg";
+import ISSvg from "@site/src/components/Icon/teleport-svg/identity-security.svg";
```

Comment:
Nit: it looks like the established convention here is that we only capitalize the first letter of an acronym (for example, we say `Svg`, not `SVG`).

So I think these should be `MwiSvg`, `IgSvg`, and `IsSvg`. Or even better, `MwiSvg`, `IdentityGovernanceSvg`, and `IdentitySecuritySvg`.